### PR TITLE
data: iriteshmishra

### PR DESCRIPTION
**Pull request by iriteshmishra**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5536"></a>

### CODE REVIEW

File: iriteshmishra.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Ritesh Mishra",
+  "displayStatsPublic": true,
```
Comment (khushalarora) [inline iriteshmishra.json:3]: Please remove this line, it is not required :+1:

File: iriteshmishra.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  "type": "personal",
+  "bio": "I am a Full Stack Developer",
+  "avatar": "https://github.com/iriteshmishra.png"
```
Comment (khushalarora) [inline iriteshmishra.json:6]: Please remove this line, it is not required :+1:

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hi @iriteshmishra, thank you for contributing :tada: 
Please rename your filename and add **` data/ `** at the beginning.
It will look like  `data/iriteshmishra.json `

**Review by iriteshmishra — COMMENTED** [review]
Changed File name

**Review by khushalarora — APPROVED** [review]
Thank you :tada:

**Review by iamthecloverly — APPROVED** [review]

File: data/iriteshmishra.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Instagram",
+      "url": "https://www.instagram.com/iriteshmishra",
+      "icon": "SiInstagram"
+    },
+    {
+      "name": "Email-Id",
+      "url": "iriteshmishra@gmail.com",
```
Comment (kumarsonsoff3) [inline data/iriteshmishra.json:33]: It should probably be `mailto:<mailID>`

```suggestion
      "url": "mailto:iriteshmishra@gmail.com",
```

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**